src/components/Toggle/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,100 @@
+import { Meta, Story } from '@storybook/react'
+import { ComponentProps } from 'react'
+import Toggle from '..'
+import Icon from '../../Icon'
+import Text from '../../Text'
+
+export default {
+  component: Toggle,
+  parameters: {
+    docs: {
+      description: {
+        component: 'Toggle button used to give an on / off state into a form.',
+      },
+    },
+  },
+  title: 'Components/Data Entry/Toggle',
+} as Meta
+
+const Template: Story<ComponentProps<typeof Toggle>> = ({
+  checked,
+  ...args
+}) => <Toggle {...args} name="Toggle" onChange={() => {}} />
```

Comment:
```suggestion
}) => <Toggle {...args} name="Toggle" />
```

Reply:
Otherwise, the code examples don't look "clean"
<img width="393" alt="CleanShot 2022-08-24 at 17 45 32@2x" src="https://user-images.githubusercontent.com/16015833/186463139-b9e44fcb-c2eb-4b2e-92d8-610f092bc2d4.png">